sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -41,6 +41,7 @@ class HttpHeaders:
 
     Authorization = "authorization"
     ETag = "etag"
+    ETagAsync = "Etag"
```

Comment:
Why is the casing different than `ETag`?

Reply:
This seems to be a quirk currently in python as far as I can tell. For async client the "etag" we get in response headers is spelled as "Etag". I added this since change feed pulls were not working properly in the async client. Only the first page would ever be received regardless of there being more because we were using "etag" instead.

Reply:
That's odd, the headers are coming from the service, why would th casing change based on the client language?

Reply:
In that case, we can then use the existing `ETag`?

Reply:
We should investigate this - as @nickcoai mentioned, these should be case insensitive. We shouldn't need to add an extra field here.
We probably need to trace this back to where the response headers are coming from. This is potentially an artifact of the different transports between sync and async (requests vs aiohttp) however in both cases the response headers should be case insensitive....

Reply:
The headers sent and received from the service are case insensitive, but python dictionary keys are not which results in this bug as we are trying to read the correct continuation value from the response header (dictionary) by using only the "etag" key

Reply:
This issue was resolved by removing the casting to dict that was being done to response headers.